lib/tasks/db_data_migrate.rake

```diff
@@ -208,6 +208,19 @@ def populate_nil_project_slugs
   puts " #{slug_count} Project slugs populated."
 end
 
+def set_default_storage_provider
+  if StorageProvider.any?
+    if StorageProvider.where(is_default: true).any?
+      $stderr.puts "0 storage_provider default statuses changed"
```

Comment:
Why are all of these using `$stderr.puts`? Seems to me that line 214 and 217 should be normal `puts` statements.

Reply:
when you run a task with the heroku cli, it will filter out data written to STDOUT.